experiments/omip_prototype/sixth_degree_omip.jl

```diff
@@ -0,0 +1,150 @@
+using ClimaOcean
+using ClimaSeaIce
+using Oceananigans
+using Oceananigans.Grids
+using Oceananigans.Units
+using Oceananigans.OrthogonalSphericalShellGrids
+using ClimaOcean.OceanSimulations
+using ClimaOcean.ECCO
+using ClimaOcean.JRA55
+using ClimaOcean.DataWrangling
+using ClimaSeaIce.SeaIceThermodynamics: IceWaterThermalEquilibrium
+using Printf
+using Dates
+using CUDA
+
+function synch!(clock1::Clock, clock2)
+    # Synchronize the clocks
+    clock1.time = clock2.time
+    clock1.iteration = clock2.iteration
+    clock1.last_Δt = clock2.last_Δt
+end
+
+synch!(model1, model2) = synch!(model1.clock, model2.clock)
+
+arch    = GPU()
+r_faces = ClimaOcean.exponential_z_faces(; Nz=60, depth=6200)
+z_faces = MutableVerticalDiscretization(r_faces)
+
+Nx = 2160 # longitudinal direction 
+Ny = 1080 # meridional direction 
+Nz = length(r_faces) - 1
+
+grid = TripolarGrid(arch;
+                    size = (Nx, Ny, Nz),
+                    z = z_faces,
+                    halo = (7, 7, 7))
+
+bottom_height = regrid_bathymetry(grid; minimum_depth=15, major_basins=1)
+grid = ImmersedBoundaryGrid(grid, GridFittedBottom(bottom_height); active_cells_map=true)
+
+#####
+##### A Propgnostic Ocean model
+#####
+
+# A very diffusive ocean
+momentum_advection = WENOVectorInvariant()
+tracer_advection   = WENO(order=7)
+
+free_surface = SplitExplicitFreeSurface(grid; substeps=70)
+closure = (ClimaOcean.OceanSimulations.default_ocean_closure(), 
+           VerticalScalarDiffusivity(κ=1e-5, ν=1e-5))
```

Comment:
```suggestion
closure = ClimaOcean.OceanSimulations.default_ocean_closure()
```

CATKE has an effective background diffusivity through its minimum TKE. We should not add any additional mixing. If you need more, you can set the minimum TKE parameter in CATKE.